### PR TITLE
feat(Form.Appearance): add theme to control size of input fields

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/Theme/info.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+For now, it only provides theming of sizes of input fields, including Autocomplete and Dropdown. See example below.
+
+You can nest `Form.Theme` to provide different themes for different parts of the form.
+
+**BETA:** The sizes are not 100% finalised and may change to be officially approved by UX through an RFC.
+
+## Usage
+
+```tsx
+import { Form } from 'gel-generic'
```
Comment (joakbjerk) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/extended-features/Form/Theme/info.mdx:18]: is `gel-generic` correct?

**Review by joakbjerk — APPROVED** [review]
Looks good! 👍 Left one comment 🙏

**Comment by tujoworker** [comment]
We had discussions about the name, where `Form.Provider` was mentioned as an alternative name. But already have the Eufemia shared `Provider` and also the `DataContext.Provider` – "Appearance" is a better naming for this purpose.

**Pull request merged**